app/components/common/ModalAPCSPContact.vue

```diff
@@ -73,13 +78,20 @@ export default Vue.extend({
       window.location.href = '#license-interest'
       this.$emit('close')
     },
+    askLocation () {
+      if (this.geo && this.geo.country) {
+        this.$v.location.$model = this.geo.country
```

Comment:
why are we doing this? Can't we set `this.location = me.get('geo')?.country` in mounted as you did with geo?

If we are setting things, we should call the function name accordingly too